subsystem/src/main/java/org/wildfly/extension/grpc/GrpcSubsystemDefinition.java

```diff
@@ -16,24 +16,54 @@
 package org.wildfly.extension.grpc;
 
 import java.util.Collection;
-import java.util.Collections;
+import java.util.List;
 
 import org.jboss.as.controller.AttributeDefinition;
 import org.jboss.as.controller.PersistentResourceDefinition;
 import org.jboss.as.controller.ReloadRequiredRemoveStepHandler;
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
 import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.operations.validation.IntRangeValidator;
+import org.jboss.as.controller.operations.validation.ModelTypeValidator;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
 
 public class GrpcSubsystemDefinition extends PersistentResourceDefinition {
 
+    static final SimpleAttributeDefinition GRPC_SERVER_HOST = new SimpleAttributeDefinitionBuilder(
+            "server-host", ModelType.STRING)
```

Comment:
I wonder if "server-host" might collide with an attribute somewhere else? Same with "server-port".

Reply:
It would only collide with something in the same subsystem. TBH what we really probably want is a `socket-binding-ref` which is a reference to `/socket-binding-group=standard-sockets/socket-binding=`.

Reply:
Ah, good to know

Reply:
Just think of it as the namespace is the subsystem path so the attributes can be named whatever.

Reply:
Everything is renamed